freqtrade/exchange/exchange.py

```diff
@@ -1444,8 +1444,9 @@ def fetch_l2_order_book(self, pair: str, limit: int = 100) -> dict:
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
-    def get_rate(self, pair: str, refresh: bool,
-                 side: EntryExit, is_short: bool) -> float:
+    def get_rate(self, pair: str, refresh: bool,  # noqa: max-complexity: 13
```

Comment:
Please don't noqa methods. If you need to add complexity, consider refactoring them to reduce complexity again. This doesn't apply only here, but also in other cases where this was added.